QEfficient/cloud/compile.py

```diff
@@ -120,7 +127,8 @@ def main(
         help="Cloud AI 100 device ids (comma-separated) e.g. [0] ",
     )
     parser.add_argument(
-        "--aic_enable_depth_first", "--aic-enable-depth-first",
+        "--aic_enable_depth_first",
```

Comment:
Not sure if `mos` and `aic_enable_depth_first` in tested for 1.14?

Reply:
Thats why default for this branch will be set to False.

Reply:
Okay, so with SDK 1.14, if I pass these options the command won't fail but might not behave correctly?
If the command will fail for SDK 1.14 we should just remove them? You can just write assert statement for it to fail in infer, compile APIs?

Reply:
No, it won't fail. This is tested.